code/src/java/pcgen/rules/persistence/CDOMControlLoader.java

```diff
@@ -126,7 +119,7 @@ private boolean parseSubLine(LoadContext context, String val, URI source)
 			return false;
 		}
 		String name = lineIdentifier.substring(colonLoc + 1);
-		if ((name == null) || name.isEmpty())
+		if (name.isEmpty())
```

Comment:
we drop it to a single condition where we can prove that the null check is useless, especially from the line right before.